<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_estimate_many_pose_similarity_params</class>
 <widget class="QWidget" name="widget_estimate_many_pose_similarity_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Set params</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Confidence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxConfidence">
       <property name="minimum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderConfidence">
       <property name="minimum">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelImages">
       <property name="text">
        <string>Images folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditImages"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonImages">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelData">
       <property name="text">
        <string>Data folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditData"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonData">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>512</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
